pages/home/serve-tools/docker-worker.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Build custom worker images with docker"
+description: "Learn how to build a custom worker image using Arcade AI"
+---
+
+import { Steps } from "nextra/components";
+
+# Build custom worker images with docker
+
+This guide shows you how to build a custom Worker image using Arcade's base Worker image. It takes you through creating a Dockerfile, installing toolkits, and running the resulting container.
+
+<Steps>
+
+### Requirements
+
+- Docker installed on your machine
+
+### Create your Dockerfile
+
+Create a Dockerfile in your project directory:
+
+```dockerfile
+ARG VERSION=latest
+
+FROM ghcr.io/arcadeai/worker-base:${VERSION}
+
+# Copy the file that lists all your desired toolkits
+COPY toolkits.txt ./
+
+# Install these toolkits
+RUN pip install -r toolkits.txt
+```
+
+With this Dockerfile, we start from the Arcade AI worker base image, copy in a file named toolkits.txt, and install each toolkit listed there.
+
+---
+
+## List Your Toolkits
+
+Create a file named toolkits.txt in the same directory. Add the toolkits you want installed:
+
+```
+arcade-google
+arcade-web
+arcade-zoom
+```
+
+Adjust this file as needed. Each line in toolkits.txt should specify a Python package name and version.
```

Comment:
What happens if I don't specify a version? There's no example of how to specify a version. Should we include that?